src/test-helpers/constants.ts

```diff
@@ -22,3 +22,9 @@ export const MAX_U8 = '255';
 
 export const MAX_I8 = '127';
 export const MIN_I8 = '-128';
+
+export const H512_U64 =
```

Comment:
```suggestion
export const MAX_H512 =
```

Reply:
I like the name suggestion!
Just changed them all to reflect this.